components/brand/AboutUs.vue

```diff
@@ -70,14 +62,9 @@
         </div>
       </div>
     </div>
-    <!-- / - mobile version -->
-    <div class="flex hidden lg:block">
-      <h4 class="uppercase text-primary font-semibold mt-32 sm:mt-12 text-[38px] leading-tight">
-        {{ pageData.fields.team }}
-      </h4>
-    </div>
+   
     <!-- First Row - Team Members -->
-    <div class="flex flex-wrap lg:flex-nowrap justify-center lg:justify-between mb-4 mt-16 hidden lg:flex">
+    <div class="flex flex-wrap lg:flex-nowrap justify-center lg:justify-between mb-4 hidden lg:flex">
```

Comment:
```suggestion
    <div class="flex-wrap lg:flex-nowrap justify-center lg:justify-between mb-4 hidden lg:flex">
```

`flex` property is being overriden by `hidden`